src/skan/test/test_csr.py

```diff
@@ -124,20 +124,20 @@ def test_multiplicity_stats():
     stats1 = csr.summarize(csr.Skeleton(skeleton0))
     stats2 = csr.summarize(csr.Skeleton(skeleton0, spacing=2))
     assert_almost_equal(
-            2 * stats1['branch-distance'].values,
-            stats2['branch-distance'].values
+            2 * stats1['branch_distance'].values,
+            stats2['branch_distance'].values
             )
     assert_almost_equal(
-            2 * stats1['euclidean-distance'].values,
-            stats2['euclidean-distance'].values
+            2 * stats1['euclidean_distance'].values,
+            stats2['euclidean_distance'].values
             )
 
 
 def test_pixel_values():
     image = np.random.random((45,))
     expected = np.mean(image)
     stats = csr.summarize(csr.Skeleton(image))
-    assert_almost_equal(stats.loc[0, 'mean-pixel-value'], expected)
+    assert_almost_equal(stats.loc[0, 'mean_pixel_value'], expected)
```

Comment:
Confused as to why the test is failing here as the variable/column name has been converted to use `_`. :thinking: